AnnoyancesFilter/sections/popups.txt

```diff
@@ -2892,15 +2868,14 @@ profile.ru#$##subscribeModal { display: none!important; }
 profile.ru#$#.modal-backdrop { display: none!important; }
 profile.ru#$#body { overflow: visible!important; }
 msc.com###newsletterSignupModal
-msc.com##.reveal-modal-bg
 mangatown.com,mangahere.cc###edShow
 coolmod.com#$#.sweet-overlay { display: none!important; }
 coolmod.com#$#.sweet-alert { display: none!important; }
 coolmod.com#$#body { overflow: visible!important; }
 5lb.ru###rees46-popup
 stroi.mos.ru##.subscribe-popup__wrapper
 iosninja.io###social-modal
-gameloft.com##.fancybox-overlay
+ciapc.com.br,pravmir.ru,jeans-meile.de,sportfm.ru,gameloft.com##.fancybox-overlay
```

Comment:
`##fancybox-overlay` and `###fancybox-content` are interrelated, so better to put them side-by-side.

Reply:
ok.